store/migrate.go

```diff
@@ -0,0 +1,37 @@
+package store
```

Comment:
Whoops. Noticed that also store/tree*.go and store/schema.go have missing copyright headers. I'll open another PR for them.